<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Konstantin &quot;konst&quot; Kucher</author>
 <class>UserForm</class>
 <widget class="QMainWindow" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSigillum</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/qsigillum.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Actions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="tbSaveResultsAs">
         <property name="toolTip">
          <string>Save results</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../res/qsigillum.qrc">
           <normaloff>:/document-save-as.png</normaloff>:/document-save-as.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbQuit">
         <property name="toolTip">
          <string>Quit</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../res/qsigillum.qrc">
           <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+Q</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Preview</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbPreview">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>409</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionPreprocessing"/>
    <addaction name="actionSegmentation"/>
    <addaction name="actionRecognition"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAboutApplication"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/configure.png</normaloff>:/configure.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/help-contents.png</normaloff>:/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAboutApplication">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
   </property>
   <property name="text">
    <string>About application</string>
   </property>
   <property name="toolTip">
    <string>About application</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/Nokia-QtCreator.png</normaloff>:/Nokia-QtCreator.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreprocessing">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/transform-scale.png</normaloff>:/transform-scale.png</iconset>
   </property>
   <property name="text">
    <string>Preprocessing</string>
   </property>
  </action>
  <action name="actionSegmentation">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/select-rectangular.png</normaloff>:/select-rectangular.png</iconset>
   </property>
   <property name="text">
    <string>Segmentation</string>
   </property>
  </action>
  <action name="actionRecognition">
   <property name="icon">
    <iconset resource="../res/qsigillum.qrc">
     <normaloff>:/draw-text.png</normaloff>:/draw-text.png</iconset>
   </property>
   <property name="text">
    <string>Recognition</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/qsigillum.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbQuit</sender>
   <signal>clicked()</signal>
   <receiver>UserForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
